plugins/pytorch/CMakeLists.txt

```diff
@@ -18,4 +18,14 @@ if( VIAME_ENABLE_PYTORCH-MMDET )
     viame/arrows/pytorch
     mmdet_trainer )
 
+  kwiver_add_python_module(
+    ${CMAKE_CURRENT_SOURCE_DIR}/netharn_detector.py
+    viame/arrows/pytorch
+    netharn_detector )
+
+  kwiver_add_python_module(
+    ${CMAKE_CURRENT_SOURCE_DIR}/_netharn_detect_predict.py
+    viame/arrows/pytorch
+    _netharn_detect_predict )
+
```

Comment:
Please put in VIAME_ENABLE_PYTORCH-NETHARN conditional

Reply:
Fixed.